### PR TITLE
changed line 61 into distance_of_time_on_words

### PR DESCRIPTION
**Pull request by barcha47**
Old line 61: distance_in_words:
New line 61 :  distance_of_time_in_words:

Fixes #0000 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: config/locales/hi-IN.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ hi-IN:
     - :month
     - :year
   datetime:
-    distance_in_words:
+    distance_of_time_in_words:
```
Comment (TildaDares) [inline config/locales/hi-IN.yml:61]: Hi @barcha47, this line should be `time_ago_in_words:`. Feel free to accept the suggestion.
```suggestion
    time_ago_in_words:
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: config/locales/hi-IN.yml (hunk 1 above)
Comment (barcha47) [reply config/locales/hi-IN.yml]: Thanks for correcting me. Hope will learn more.

**Comment by barcha47** [comment]
Hi @TildaDares , please correct me why tests/ system-tests(pull-request) failing again.

**Review by TildaDares — APPROVED** [review]
Thanks @barcha47!! Great job.

**Comment by TildaDares** [comment]
Congratulations 🎉

**Pull request merged**